dependencies.yaml

```diff
@@ -119,6 +119,7 @@ main_requires:
   perl(FindBin):
   perl(Getopt::Long::Descriptive):
   perl(IO::Handle):
+  perl(IO::Uncompress::UnXz):
```

Comment:
A comment why we need this would be useful.

Reply:
Done